Content/WebPage/WebPageViewController.swift

```diff
@@ -10,44 +10,40 @@ import UIKit
 import WebKit
 
 public protocol WebPageViewControllerDelegate: AnyObject {
-    func webViewCanGoBack()
-    func webViewCanGoForward()
-    func webViewEstimatedProgressChanged(_ progress: Double)
-    func urlForNavigationAction(_ url: URL?)
-    func webViewDidStartProvisionalNavigation()
-    func webViewDidFinishNavigation()
+    func getError(_: Error)
 }
 
-public class WebPageViewController: UIViewController {
+open class WebPageViewController: UIViewController {
```

Comment:
The reason for making this class `open` is to provide the flexibility of the customisation to the apps.